ovirt_imageio/client/_api.py

```diff
@@ -20,16 +20,17 @@
 from urllib.parse import urlparse
 
 from .. _internal import blkhash
-from .. _internal import io
 from .. _internal import qemu_img
 from .. _internal import qemu_nbd
 from .. _internal.backends import http, nbd
 from .. _internal.handlers import checksum as _checksum
 from .. _internal.nbd import UnixAddress
 
+from . import _io
+
 # API constants.
-BUFFER_SIZE = io.BUFFER_SIZE
-MAX_WORKERS = io.MAX_WORKERS
+BUFFER_SIZE = _io.BUFFER_SIZE
+MAX_WORKERS = _io.MAX_WORKERS
```

Comment:
With the change these duplicates can be omitted.

Reply:
Why, we still export these names

Reply:
Uhm, because they can now be exported directly from `_io` in the `__init__` module. But I guess the intend is to have the `_api` to expose the names.

It's fine.